src/@types/global.d.ts

```diff
@@ -92,6 +92,7 @@ declare global {
         mxUIStore: UIStore;
         mxSetupEncryptionStore?: SetupEncryptionStore;
         mxRoomScrollStateStore?: RoomScrollStateStore;
+        grecaptcha: any;
```

Comment:
This is now specified lower and shouldn't be needed anymore https://github.com/matrix-org/matrix-react-sdk/pull/6412/files#diff-af565283322a1965b4ea9bfb48fe1f578631a934aadbfc7a413fe0d5e84746fbR194